tests/pages/base_page.py

```diff
@@ -3,7 +3,9 @@
 
 sys.path.append(os.path.dirname(__file__))
```

Comment:
I think you can remove all calls to `sys.path.append(os.path.dirname(__file__))`.